src/sys/ptrace/linux.rs

```diff
@@ -173,6 +173,7 @@ libc_bitflags! {
     since="0.10.0",
     note="usages of `ptrace()` should be replaced with the specialized helper functions instead"
 )]
+#[allow(clippy::missing_safety_doc)]    // It's deprecated anyway
```

Comment:
Any reason we shouldn't just remove this API at this point? It's been 2.5 years at this point. I'd imagine if we've had at least two releases and it's been at least a year we can probably remove deprecated APIs.

Reply:
Good point.  I'll take it out.

Reply:
Ok, I removed it.  But I decided to remove some other stuff, too, and created a new PR: #1255 .